synapse/lib/stormhttp.py

```diff
@@ -32,24 +32,14 @@ async def _httpEasyGet(self, url, headers=None, ssl_verify=True):
 
             ssl_verify (bool): Perform SSL/TLS verification. Defaults to true.
 
+            params (dict): Optional parameters which may be passed to the request.
```

Comment:
Since we're passing the params directly through to aiohttp, technically this can be multiple types and I've got tests for each of them; if only to 1) avoid isinstance code and 2) support recurring key options in params